app/src/main/java/com/codesoom/assignment/application/TaskService.java

```diff
@@ -33,20 +43,35 @@ public Task createTask(Task source) {
         return task;
     }
 
+    /**
+     * 할 일을 수정하고, 수정된 할 일을 리턴합니다.
+     * @param id 수정할 할 일의 id
+     * @param source 수정할 내용
+     * @return 수정이 완료된 할 일
+     */
     public Task updateTask(Long id, Task source) {
         Task task = getTask(id);
         task.setTitle(source.getTitle());
 
         return task;
     }
 
+    /**
+     * 할 일을 삭제하고, 삭제된 할 일을 리턴합니다.
+     * @param id 삭제ㄷ할 할 일의 아이디
+     * @return 삭제된 할 일
+     */
     public Task deleteTask(Long id) {
         Task task = getTask(id);
         tasks.remove(task);
 
         return task;
     }
 
+    /**
+     * 아이디 값을 증가시킵니다.
+     * @return 증가된 아이디 값
+     */
```

Comment:
메소드 이름과의 불일치에 주의해 주세요.
```suggestion
    /**
     * 아이디 값을 생성해 리턴합니다.
     * @return 새로운 아이디 값
     */
```

Reply:
수정했습니다